btk/metrics.py

```diff
@@ -1,484 +1,110 @@
-    true_table["num_detections2"] = np.zeros(num_true, dtype=int)
-    # detection id of closest detection with 2 algorithms [0 - num_det]
-    true_table["closest_det_id1"] = np.ones(num_true, dtype=int) * -1.0
-    true_table["closest_det_id2"] = np.ones(num_true, dtype=int) * -1.0
-    # difference in centroids, i band magnitude and redshift between an object
-    # and its match with algorithm 2
-    true_table["dm_match"] = np.zeros(num_true, dtype=int)
-    true_table["dz_match"] = np.zeros(num_true, dtype=int)
-    true_table["ddist_match"] = np.zeros(num_true, dtype=int)
-    true_table["dnorm_dist_match"] = np.zeros(num_true, dtype=int)
-    # find distance to nearest neighbor. If isolated then set to np.inf
-    true_table["min_dist"] = np.ones(num_true) * np.inf
-    get_closest_neighbor_distance(true_table)
-    # initialize detected objects table columns
-    num_det = len(detected_table)
-    detected_table["detection_id"] = range(num_det)  # id in blend [0-num_det]
-    # index of blend in test size [0 - len(blend_summary)]
-    detected_table["blend_index"] = np.ones(num_det, dtype=int) * (
-        batch_index * batch_size + blend_index
-    )
-    # index of batch in test size [0 - batch_size]
-    detected_table["batch_index"] = np.ones(num_det, dtype=int) * batch_index
-    # id of closest true object; [0 - num_true] if detected, else -1
-    detected_table["match_true_id1"] = np.ones(num_det, dtype=int) * -1
-    detected_table["match_true_id2"] = np.ones(num_det, dtype=int) * -1
-    detected_table["match_galtileid1"] = np.ones(num_det, dtype=int) * -1
-    detected_table["match_galtileid2"] = np.ones(num_det, dtype=int) * -1
+import skimage.metrics
 
 
 def get_detection_match(true_table, detected_table):
     """Match detections to true objects and update values in the input
     blend catalog and detection catalog.
-
     Function does not return anything, only the astropy tables are updated.
-
     Args:
         true_table (astropy.table.Table): Table with entries corresponding to
             the true object parameter values in one blend.
         detected_table(astropy.table.Table): Table with entries corresponding
             to output of measurement algorithm in one blend.
     """
+    match_table = astropy.table.Table()
     if len(detected_table) == 0 or len(true_table) == 0:
         # No match since either no detection or no true objects
         return
     t_x = true_table["x_peak"][:, np.newaxis] - detected_table["x_peak"]
     t_y = true_table["y_peak"][:, np.newaxis] - detected_table["y_peak"]
     dist = np.hypot(t_x, t_y)
-    norm_size = true_table["size"]
-    norm_dist = dist / norm_size[:, np.newaxis]
-    detected_table["dSigma_min"] = np.min(norm_dist, axis=0)
     detected_table["d_min"] = np.min(dist, axis=0)
-    detection_threshold1 = 5
-    condlist1 = [
-        np.min(dist, axis=0) <= detection_threshold1,
-        np.min(dist, axis=0) > detection_threshold1,
+    detection_threshold = 5
```

Comment:
We probably want to make this changeable by the user.  Maybe even allow it to be a function in case people want to do things like have it in units of the size of the galaxy

Reply:
Yes, we definitely want more flexibility in how the matches are done in the future, will get to it later.

Reply:
sounds good, maybe users can provide a function that calculates the distance based on the values of `x_peak` and `y_peak`, need to think more about how exactly